src/libraries/System.Net.Http/src/System/Net/Http/HttpClient.CreateDefaultHandler.cs

```diff
@@ -0,0 +1,13 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.Net.Http
+{
+    public partial class HttpClient
+    {
+        private static HttpMessageHandler CreateDefaultHandler()
```

Comment:
@karelz @stephentoub should we also hook up WinHttpHandler as part of this PR?

Reply:
No, I think we should keep `SocketsHttpHandler` as primary on all platforms where we can.
While `WinHttpHandler` has its usages in rare/extreme cases, we do not expect almost anyone to use it.
cc @geoffkizer

Reply:
I didn't mean to change the default, just hook it up with a feature switch. I was just surprised to see `360 440` WinHttpHandler nuget downloads since November.